plugins/catalog-backend-module-ldap/src/processors/LdapOrgEntityProvider.ts

```diff
@@ -89,6 +89,11 @@ export interface LdapOrgEntityProviderOptions {
    * The function that transforms a group entry in LDAP to an entity.
    */
   groupTransformer?: GroupTransformer;
+
+  /**
+   * Specify which vendor to use.
+   */
+  vendorOverride?: String;
```

Comment:
All string types should be lowercase `string`. But would it make more sense to be able to pass in an actual vendor instance here instead? Then the package could export

```ts
export const vendors = Object.freeze({
  microsoft: (): LdapVendor => ...,
});
```

that way people can easily add their completely own ones before waiting for us to do it in open source and then waiting for a release to happen